<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SHHouseCtlDlg</class>
 <widget class="QWidget" name="SHHouseCtlDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,14,1">
       <item>
        <widget class="QLabel" name="m_lbTitle">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>家居控制</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>708</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,3">
       <item>
        <widget class="QListWidget" name="m_listwgRoom">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="m_listwgDevice">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="m_scrollAreaDevice">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="m_scrollAreaWgDevice">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>554</width>
            <height>491</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
